src/requirements.yml

```diff
@@ -8,12 +8,17 @@
   name: cloudwatch_agent
 - src: https://github.com/cisagov/ansible-role-dev-ssh-access
   name: dev_ssh_access
+- src: https://github.com/cisagov/ansible-role-devops-user
+  name: devops
 - src: https://github.com/cisagov/ansible-role-freeipa-client
   name: freeipa_client
 - src: https://github.com/cisagov/ansible-role-guacamole
   name: guacamole
+  version: feature/kerberization
```

Comment:
🎗️ Usual reminder to remove this after https://github.com/cisagov/ansible-role-guacamole/pull/6 is merged.

Reply:
See commit e80b5bc.